_includes/documents-collection.html

```diff
@@ -1,16 +1,14 @@
 {% assign entries = site[include.collection] %}
 
-{% if include.sort_by == 'title' %}
+{% if include.sort_by %}
   {% if include.sort_order == 'reverse' %}
-    {% assign entries = entries | sort: 'title' | reverse %}
+    {% assign entries = entries | sort: include.sort_by | reverse %}
   {% else %}
-    {% assign entries = entries | sort: 'title' %}
+    {% assign entries = entries | sort: include.sort_by %}
   {% endif %}
-{% elsif include.sort_by == 'date' %}
+{% else %}
   {% if include.sort_order == 'reverse' %}
```

Comment:
Looks like we're doing repeated work here - we only need to check for `sort_order` once, after the `sort_by` check. Would you help fix this as well?

Reply:
Don't we need to be able to use the `sort_by` parameter even if `sort_order` is not specified?

Reply:
To be exact, here's what we have for now:

```javascript
if (sort_by is given) {
  if (reverse) {
    entries = entries | sort_by | reverse;
  } else {
    entries = entries | sort_by;
  }
} else {
  if (reverse) {
    entries = entries | reverse;
  }
}
```

And here's what I'm expecting:

```javascript
if (sort_by is given) {
  entries = entries | sort_by;
}

if (reverse) {
  entries = entries | reverse;
}
```

Clearly the latter looks more concise, right?

Reply:
You're right, I have made the changes